mqtt/mqtt3-fuzz/Cargo.toml

```diff
@@ -9,13 +9,13 @@ publish = false
 [dependencies]
```

Comment:
do we still use this one? :)

Reply:
it fuzzes protocol codec, so why not have it?